opentelemetry-semantic-conventions/scripts/templates/registry/rust/resource.rs.j2

```diff
@@ -1,21 +1,39 @@
-//! # Resource Semantic Conventions
-//!
-//! The [resource semantic conventions] define a set of standardized attributes
-//! to be used in `Resource`s.
-//!
-//! [resource semantic conventions]: https://github.com/open-telemetry/semantic-conventions/tree/main/model/resource
-//!
-//! ## Usage
-//!
-//! ```rust
-//! use opentelemetry::KeyValue;
-//! use opentelemetry_sdk::{trace::{config, TracerProvider}, Resource};
-//! use opentelemetry_semantic_conventions as semconv;
-//!
-//! let _tracer = TracerProvider::builder()
-//!     .with_config(config().with_resource(Resource::new(vec![
-//!         KeyValue::new(semconv::resource::SERVICE_NAME, "my-service"),
-//!         KeyValue::new(semconv::resource::SERVICE_NAMESPACE, "my-namespace"),
-//!     ])))
-//!     .build();
-//! ```
+{%- import 'macros.j2' as attr_macros -%}
+// DO NOT EDIT, this is an auto-generated file
+//
+// If you want to update the file:
+// - Edit the template at scripts/templates/registry/rust/attributes.rs.j2
```

Comment:
there is no file with this name?

Reply:
Fixed in [4970ee9](https://github.com/open-telemetry/opentelemetry-rust/pull/2098/commits/4970ee9314ca5cb72376b21bb807ec14da0d4cc8)